src/megameklab/com/printing/PrintEntity.java

```diff
@@ -127,7 +128,7 @@ protected void printImage(Graphics2D g2d, PageFormat pageFormat, int pageNum) {
         drawArmor();
         drawStructure();
         Element eqRect = getSVGDocument().getElementById("inventory");
-        if ((null != eqRect) && (eqRect instanceof SVGRectElement)) {
+        if (eqRect instanceof SVGRectElement) {
```

Comment:
Is it possible for eqRect to be null?

Reply:
Yes, but in that case `eqRect instanceof SVGRectElement` will be false. So the null check is redundant.

Reply:
And by "possible" I mean "only if someone screws around with the template." And I'm including myself in that "someone."

Reply:
Makes sense.